docs/source/tts/data/ngc_models_codec.csv

```diff
@@ -4,3 +4,4 @@ mel_codec_22khz_medium,LibriVox and Common Voice,22050Hz,nemo.collections.tts.mo
 mel_codec_44khz_medium,LibriVox and Common Voice,44100Hz,nemo.collections.tts.models.AudioCodecModel,`mel_codec_44khz_medium <https://ngc.nvidia.com/catalog/models/nvidia:nemo:mel_codec_44khz_medium>`_,``https://api.ngc.nvidia.com/v2/models/nvidia/nemo/mel_codec_44khz_medium/versions/v1/files/mel_codec_44khz_medium.nemo``
 mel_codec_22khz_fullband_medium,LibriVox and Common Voice,22050Hz,nemo.collections.tts.models.AudioCodecModel,`mel_codec_22khz_fullband_medium <https://ngc.nvidia.com/catalog/models/nvidia:nemo:mel_codec_22khz_fullband_medium>`_,``https://api.ngc.nvidia.com/v2/models/nvidia/nemo/mel_codec_22khz_fullband_medium/versions/v1/files/mel_codec_22khz_fullband_medium.nemo``
 mel_codec_44khz_fullband_medium,LibriVox and Common Voice,44100Hz,nemo.collections.tts.models.AudioCodecModel,`mel_codec_44khz_fullband_medium <https://ngc.nvidia.com/catalog/models/nvidia:nemo:mel_codec_44khz_fullband_medium>`_,``https://api.ngc.nvidia.com/v2/models/nvidia/nemo/mel_codec_44khz_fullband_medium/versions/v1/files/mel_codec_44khz_fullband_medium.nemo``
+audio_codec_low_frame_rate_22khz,LibriVox and Common Voice,22050Hz,nemo.collections.tts.models.AudioCodecModel,`audio_codec_low_frame_rate_22khz <https://huggingface.co/nvidia/Low-Frame-rate-Speech-Codec>`_,``https://huggingface.co/nvidia/Low-Frame-rate-Speech-Codec/resolve/main/Low_Frame-rate_Speech_Codec_21Hz.nemo``
```

Comment:
Should we remove "ngc" from the name of this csv file? Or create a `huggingface_models_codec.csv`? It looks like this file is only referenced in `checkpoints.rst`.

Reply:
I removed "ngc" from the csv file.